pyomo/contrib/doe/documentation/docs/source/index.rst

```diff
@@ -0,0 +1,453 @@
+where:
+
+*  :math:`\boldsymbol{\varphi}` are design variables, which are manipulated to maximize the information content of experiments. It should consist one or more of  :math:`\mathbf{u}(t), \mathbf{y}^{\mathbf{0}}({t_0}),\overline{\mathbf{w}}`. With a proper model formulation, the timepoints for control or measurements :math:`\mathbf{t}` can also be degree of freedoms.
+*  :math:`\mathbf{M}` is Fisher information matrix (FIM), estimated as the inverse of the covariance matrix of parameter estimates  :math:`\boldsymbol{\hat{\theta}}`. A large FIM indicates more information contained in the experiment for parameter estimation.
+*  :math:`\mathbf{Q}` is dynamic sensitivity matrix, containing the partial derivatives of  :math:`\mathbf{y}` with respect to  :math:`\boldsymbol{\theta}`.
+*  :math:`\Psi` is the design criteria to measure FIM.
+*  :math:`\mathbf{V}_{\boldsymbol{\theta}}(\boldsymbol{\hat{\theta}})^{-1}` is the FIM of previous experiments.
+
+Pyomo.DOE provides four design criteria  :math:`\Psi` to measure the size of FIM:
+
+.. list-table:: Pyomo.DOE design criteria
+    :header-rows: 1
+    :class: tight-table
+
+    * - Design criterion
+      - Computation
+      - Geometrical meaning
+    * - A-optimality
+      -   :math:`\text{trace}({\mathbf{M}})`
+      - Dimensions of the enclosing box of the confidence ellipse
+    * - D-optimality
+      -   :math:`\text{det}({\mathbf{M}})`
+      - Volume of the confidence ellipse
+    * - E-optimality
+      -   :math:`\text{min eig}({\mathbf{M}})`
+      - Size of the longest axis of the confidence ellipse
+    * - Modified E-optimality
+      -   :math:`\text{cond}({\mathbf{M}})`
+      - Ratio of the longest axis to the shortest axis of the confidence ellipse
+
+In order to solve problems of the above, Pyomo.DOE implements the 2-stage stochastic program. Please see Wang and Dowling (2022) for details.
+
+Pyomo.DOE Required Inputs
+--------------------------------
+The required inputs to the Pyomo.DOE solver are the following:
+
+* A function that creates the process model
+* Dictionary of parameters and their nominal value
+* Dictionary of measurements and their measurement time points
+* Dictionary of design variables and their control time points
+* Prior FIM
+* Subordinate local and global NLP optimization solver
+
+Below is a list of arguments that Pyomo.DOE expects the user to provide.
+
+param_init : ``dictionary``
+    A ``dictionary`` of parameter names and values. If they are an indexed variable, put the variable name and index in a nested ``Dictionary``.
+
+design_variable_timepoints : ``dictionary``
+    A ``dictionary`` of design variable names and its control time points. If this design var is independent of time (constant), set the time to [0]
+
+measurement_object : ``object``
+    A ``object`` of the measurements, provided by the measurement class.
+
+
+create_model : ``function``
+    A ``function`` representing a deterministic process model.
+
+
+prior_FIM : ``list``
```

Comment:
How is the matrix represented as a list?

Reply:
checked. it is numpy array.